README.md

```diff
@@ -15,9 +15,9 @@ There are four ways to install TruffleRuby releases and nightly builds:
   to use TruffleRuby inside GraalVM.
 
 * Via your [Ruby manager/installer](doc/user/ruby-managers.md) (RVM, rbenv,
-  chruby, ruby-build, ruby-install). This contains only TruffleRuby, in the
+  chruby, ruby-build, ruby-install). This contains TruffleRuby only in the
   [Native configuration](#truffleruby-runtime-configurations), making it a smaller
-  download. It is meant for users who only want a Ruby implementation and are already
```

Comment:
The wording here originally intends "TruffleRuby" to mean "the TruffleRuby standalone distribution" in contrast to GraalVM (which is always a JDK + the GraalVM Compiler + additional components).
I agree the wording is not clear though.

Reply:
Do you want to change it?

Reply:
Yes, I'll try to clarify this section, it might need some reorganization as well.

I'll close this PR since I'll do the change, but thanks for opening it.